config/routes.rb

```diff
@@ -3,4 +3,11 @@
 
   # Defines the root path route ("/")
   # root "articles#index"
+
+  get "/guests/:id", to: "guests#show", as: :show_guest
+  patch "/guests/:id", to: "guests#update", as: :patch_suite
```

Comment:
A more RESTful option for this route could be:
- post "/reservations", to: "reservations#create" 
OR
- post "/guests/:guest_id/rooms", to: "reservations#create" if you don't want to pass the guest ID as a hidden field